### PR TITLE
Refs RHIROS_346 - make modifications to custom metrics

### PR DESCRIPTION
**Pull request by kgaikwad**

### CODE REVIEW

File: ros/processor/insights_engine_result_consumer.py [hunk 1]
```diff
@@ -53,15 +53,13 @@ def run(self):
         for msg in iter(self):
             if msg.error():
                 print(msg.error())
```
Comment (r14chandra) [inline ros/processor/insights_engine_result_consumer.py:55]: @kgaikwad Just noticed one thing. Is this print statement needed here or should we use `log`?

Comment (kgaikwad) [reply ros/processor/insights_engine_result_consumer.py]: @r14chandra  - Fixed! Replaced with log statement. Created [bug](https://issues.redhat.com/browse/RHIROS-456) based on my observation.

**Review by r14chandra — APPROVED** [review]
LGTM 👍🏻

**Review by saltgen — APPROVED** [review]
@kgaikwad Changes look good to me :+1: 
Additionally can we include a few happy path unittests, for instance one each for counters,

- processor_requests_success
- processor_requests_failures
- kafka_failures

**Comment by kgaikwad** [comment]
> @kgaikwad Changes look good to me +1 Additionally can we include a few happy path unittests, for instance one each for counters,
> 
>     * processor_requests_success
> 
>     * processor_requests_failures
> 
>     * kafka_failures

Hello @saltgen - I tried to tests by referring [python client plugin](https://github.com/prometheus/client_python) but facing few issues and need to look into it closely. So instead of blocking this change, could you open a new JIRA for adding these tests and assigned to me?

**Comment by saltgen** [comment]
> > @kgaikwad Changes look good to me +1 Additionally can we include a few happy path unittests, for instance one each for counters,
> > ```
> > * processor_requests_success
> > 
> > * processor_requests_failures
> > 
> > * kafka_failures
> > ```
> 
> Hello @saltgen - I tried to tests by referring [python client plugin](https://github.com/prometheus/client_python) but facing few issues and need to look into it closely. So instead of blocking this change, could you open a new JIRA for adding these tests and assigned to me?

Sounds good to me :ok_hand: Created the new JIRA [here](https://issues.redhat.com/browse/RHIROS-468)

**Pull request merged**